src/pages/en/tutorial/6-islands/1.md

```diff
@@ -121,17 +120,17 @@ import SvelteCounter from '../components/SvelteCounter.svelte';
 </BaseLayout>
 ```
 
-1. Which of the five components will be **hydrated islands**, sending JavaScript to the client that will run even after the site has been built?
+1. Which of the five components will be **hydrated** islands, sending JavaScript to the client?
 
     || `<PreactBanner client:load />` and `<SvelteCounter client:visible />` will be hydrated islands. ||
 
 2. In what way(s) will the two `<PreactBanner />` components be the same? In what way(s) will they be different? 
 
-    || Same: They both render the same HTML elements and look the same. Different: One will be generated at build time and will not rerender until your site is rebuilt. It will look the same until your site is rebuilt. The one with a client directive will execute its JavaScript on every page refresh, showing an updated component. ||
+  || **Same**: They both show the same HTML elements and look the same initially. Different: The component with the `client:load` directive will rerender after the page is loaded, and any interactive elements that it has will work. ||
 
-3. If you couldn't see your website's code, only the live published page, how would you tell which `<SvelteCounter />` component used `client:visible`? 
+3. Assume the `SvelteCounter` component shows a number and has a button to increase it. If you couldn't see your website's code, only the live published page, how would you tell which of the two `<SvelteCounter />` component used `client:load`? 
```

Comment:
If you want to change this to `client:load` then be sure to change it in the code sample too! It doesn't look like that was changed.

Reply:
Oops, I'd experimented with that, but decided to keep it the way it is! Fixed

Reply:
Did you fix this? It didn't change here.  😅

Reply:
🤦 should be changed now!